hypothesis-python/tests/quality/test_integers.py

```diff
@@ -61,7 +60,7 @@ def problems(draw):
 @given(problems())
 def test_always_reduces_integers_to_smallest_suitable_sizes(problem):
     n, blob = problem
-    blob = hbytes(blob)
+    blob = bytes(blob)
```

Comment:
Do we actually need this cast to `bytes()`? All the examples are byte strings, and the second argument of `problems()` is already cast to `bytes()` in the strategy definition.

Reply:
Well spotted!  Grepping for cases where we cast a `.buffer` to bytes turns up a few other cases too :smile: